doc/Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-TI-CC3200/index.md

```diff
@@ -130,24 +164,28 @@ make
 
 Refer to [C SDK Linux page]({{root_url}}Programming-guide/Using-Kaa-endpoint-SDKs/C/SDK-Linux/) for more details.
 
-### Building C SDK on Windows
+</div><div id="windows2" class="tab-pane fade" markdown="1" >
 
-Open the Cygwin terminal and execute the following:
+Open the Cygwin terminal and run:
 
-```
+```bash
 mkdir -p build
 cd build
 cmake.exe -G "Unix Makefiles" -DKAA_PLATFORM=cc32xx -DCC32XX_TOOLCHAIN_PATH=c:/cygwin/opt/kaa -DCMAKE_TOOLCHAIN_FILE=../toolchains/cc32xx.cmake -DBUILD_TESTING=OFF ..
```

Comment:
Hi rasendubi,

Thanks for reply,

the errors be attached below,
[ouput log.txt](https://github.com/kaaproject/kaa/files/1033327/ouput.log.txt)